### ISSUE TITLE
Error is logged when command be written to closed sequencer

### ISSUE THREAD

**Issue by Zelldon** [camunda/zeebe#13021]
**Describe the bug**

Looks like when we step down or become inactive we close our resources in the wrong order. 

We see several error logs with: `Unexpected error on writing CREATE command Failed to write request to logstream`

Interesting is actually the log statement before, which says actually why: `Rejecting write of [LogAppendEntryImpl[key=-1, sourceIndex=-1, recordMetadata=RecordMetadata{recordType=COMMAND, valueType=PROCESS_INSTANCE_CREATION, intent=CREATE}, recordValue={"bpmnProcessId":"one-task-one-timer","processDefinitionKey":0,"processInstanceKey":-1,"version":-1,"variables":"gah1bmlxdWVJZNkkZjJiMmMxMWQtZjg1YS00OWUxLWFlN2QtYTI4MWY5YWVhYzNk","fetchVariables":[],"startInstructions":[]}]], sequencer is closed
`

Error group: https://console.cloud.google.com/errors/detail/CNy21ZD_7pnW3AE;service=zeebe;time=P7D?project=camunda-saas-int-chaos

**To Reproduce**
Not sure, restart a node?

**Expected behavior**

Is it a problem than combine both logs to make the error more clear, and have a why it happens. For me it looks like more a warning not an error.

**Log/Stacktrace**

[FULL LOG](https://console.cloud.google.com/logs/query;query=%0AlogName:%22stdout%22%0Aresource.type%3D%22k8s_container%22%0Aresource.labels.container_name%3D%22zeebe%22%0Aresource.labels.pod_name%3D%22zeebe-2%22%0Aresource.labels.cluster_name%3D%22worker-chaos-1%22%0Aresource.labels.project_id%3D%22camunda-saas-int-chaos%22%0Aresource.labels.location%3D%22europe-west1%22%0Aresource.labels.namespace_name%3D%22e5b261ca-ae75-4fad-9274-bd0be628f088-zeebe%22;timeRange=2023-06-03T09:46:09.117Z%2F2023-06-03T10:46:09.117Z;pinnedLogId=2023-06-03T10:15:37.957328533Z%2Fvw51595halye85ji;cursorTimestamp=2023-06-03T10:15:38.017349292Z?project=camunda-saas-int-chaos)

<details><summary>Full Stacktrace</summary>
 <p>

```
Unexpected error on writing CREATE command Failed to write request to logstream
```

</p>
</details>

**Environment:**
- OS: saas
- Zeebe Version:[zeebe:8.3.0-SNAPSHOT-main-7ad25a56](https://console.cloud.google.com/errors/detail/CNy21ZD_7pnW3AE;service=zeebe;version=8.3.0-SNAPSHOT-main-7ad25a56;time=P7D?project=camunda-saas-int-chaos) 
- Configuration:

**Comment by deepthidevaki** [comment]
Duplicate of https://github.com/camunda/zeebe/issues/12780 Already fixed.

**Comment by Zelldon** [comment]
@deepthidevaki are you sure? 

The [PR](https://github.com/camunda/zeebe/pull/12910) was merged 01.06. 09:14 am
The error appeared last time [03/06/2023, 12:15 with snapshot](https://console.cloud.google.com/logs/query;query=error_group%2528%22CNy21ZD_7pnW3AE%22%2529%0AlogName:%22stdout%22%0Aresource.type%3D%22k8s_container%22%0Aresource.labels.container_name%3D%22zeebe%22%0Aresource.labels.namespace_name%3D%22e5b261ca-ae75-4fad-9274-bd0be628f088-zeebe%22%0Aresource.labels.project_id%3D%22camunda-saas-int-chaos%22%0Aresource.labels.pod_name%3D%22zeebe-2%22%0Aresource.labels.cluster_name%3D%22worker-chaos-1%22%0Aresource.labels.location%3D%22europe-west1%22;timeRange=2023-06-03T09:46:09.097Z%2F2023-06-03T10:46:09.097Z;cursorTimestamp=2023-06-03T10:15:39.117462108Z?project=camunda-saas-int-chaos) :thinking: 

Of course it depends when the cluster was created. 



---------



Ok seems the cluster existed already at [31.05](https://console.cloud.google.com/logs/query;query=%0AlogName:%22stdout%22%0Aresource.type%3D%22k8s_container%22%0Aresource.labels.container_name%3D%22zeebe%22%0Aresource.labels.namespace_name%3D%22e5b261ca-ae75-4fad-9274-bd0be628f088-zeebe%22%0Aresource.labels.project_id%3D%22camunda-saas-int-chaos%22%0Aresource.labels.pod_name%3D%22zeebe-2%22%0Aresource.labels.cluster_name%3D%22worker-chaos-1%22%0Aresource.labels.location%3D%22europe-west1%22;timeRange=2023-05-31T09:46:09.097Z%2F2023-06-03T10:46:09.097Z;pinnedLogId=2023-05-31T10:57:36.333585887Z%2Fa0rwuyo6g2o6grj2;cursorTimestamp=2023-05-31T10:59:06.075404320Z?project=camunda-saas-int-chaos) :detective: @deepthidevaki is right. :white_check_mark: 

Sorry for the noise.
